### DISCUSSION TITLE
improve variants section on product creation

### DISCUSSION THREAD

**Discussion by wychoong** [lunarphp/lunar#1048] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Currently product variants creation requires "save" and "queued job" before it populates.

**Describe the solution you'd like**
- doesn't require save to populate the variant table
- able to fill/edit minimal variant info (sku, price, stock, backorder, purchasablity) in base product page

example: https://youtu.be/qukulIfqtjs?t=136

**Describe alternatives you've considered**

**Additional context**

**Comment by wychoong** [comment]
![image](https://user-images.githubusercontent.com/67364036/210799300-db2660f2-ce94-4d46-b16e-6b4f63004b5c.png)

@glennjacobs @markmead any thoughts? this screen is during the product creation, edit will be pretty much the same. if this design is good to go then I can work on the backend logic to save the product and also a PR

**Comment by glennjacobs** [comment]
I think I would be happy to see a PR on this. We need to consider the SKU generation and validation config.

@alecritson do you agree?
